.github/labeler.yml

```diff
@@ -0,0 +1,25 @@
+# https://github.com/actions/labeler#common-examples
+# Adapted from https://github.com/rapidsai/clx/blob/main/.github/CODEOWNERS
+# Labels culled from https://github.com/rapidsai/clx/labels
+
+Python:
+  - 'python/**'
+  - 'notebooks/**'
+  
+libclx:
+  - 'cpp/**'
+
+CMake:
+  - '**/CMakeLists.txt'
+  - '**/cmake/**'
+  
+Java:
+  - 'java/**'
```

Comment:
This section should match this repo's labels and the contents of the .github/CODEOWNERS file as a starting guide - PICs in this repo would then add or suggest more labels as needed